test/spec.yml

```diff
@@ -457,6 +457,47 @@
         exit: 1
         err: "Empty string env 'EMPTY' in binaris.yml is not supported"
 
+- test: "concurrent execution model: deploy invoke multiple times"
+  steps:
+    -   in: |-
+            cat <<EOF >/home/dockeruser/test/binaris.yml
```

Comment:
I'm not sure why we started writing tests with the full absolute path but these tests run from the `test` directory and do not need it. Is there an advantage to this method?

Reply:
Good point @rylandg